src/HeaderBar/Apps.js

```diff
@@ -66,7 +69,7 @@ function Search({ value, onChange, onIconClick, contextPath }) {
             </span>
 
             <span>
-                <a href={`${contextPath}/dhis-web-menu-management`}>
+                <a href={`${baseUrl}dhis-web-menu-management`}>
```

Comment:
I would prefer to copy the `joinPath` helper for now (until exposed by app-runtime).
This seems too brittle and could break instances.

Reply:
Agreed, will do